distributions/openhab-online/readme.md

```diff
@@ -0,0 +1,37 @@
+openHAB online distro
+---------------------
+
+
+The online distro doesn't contain many features.
+You'll need to install those manually for now.
+
+
+1. start openhab
+	start.sh
+
+2. view available features
+	feature:list
+
+3. install a feature e.g. install rest docs
+	feature:install openhab-misc-restdocs
```

Comment:
I think we need to discuss how to deal with "minimal" vs. "online" distro.
For me, the online distro should result in the same functional setup as the offline distro - the only difference is the fact that the one contains the artefacts locally, so that they can be installed from there, while the other uses a remote repo.

I am also thinking about how to best handle the "openhab-demo" feature installation - at the moment, people have to do it manually in the shell, which is definitely not an easy start, so we need to change this.
I was wondering about having a config option in the addons.cfg (where you can also specify which bindings etc. should be installed), something like "runtime = minimal", "runtime = normal", "runtime = demo". The distro could then always come as minimal and install at first startup additional features (with the benefit that you can also strip it down again by changing this configuration value). This would be exactly the same for online and offline distro. What do you think?